util/chplenv/utils.py

```diff
@@ -45,37 +45,30 @@ def try_run_command(command, stdout=True, stderr=False, cmd_input=None):
                                    stderr=subprocess.PIPE,
                                    stdin=subprocess.PIPE)
     except OSError:
-        return False, 0, ''
+        return False, 0, None, None
```

Comment:
If I were writing this, I'd make this return a tuple explicitly to be consistent with the other return:

```suggestion
        return (False, 0, None, None)
```